docs/content/guides/developer/app-examples/reviews-rating.mdx

```diff
@@ -0,0 +1,451 @@
+---
+title: Reviews Rating
```

Comment:
nit: The title reads a little strangely to me, either "Reviews and Ratings" or "Review Rating" read better.